2bf8293e8fec0aa7efb450a7bf1a14633d16abcd..04762b357635d3adb85ccd5e3456e81c23033394
50fdd5e2b5c6347ccffb19efe31f271feca92de0..ce5477ef02688e4f535c6916c26193743ab38f40

commit 2b16e1fa6161dff3e8f38b11327842b24733fc7f
Author:     Robert Walker <bob.mt.wya@gmail.com>
AuthorDate: Tue Jan 31 15:58:00 2023 +0000
Commit:     Robert Walker <bob.mt.wya@gmail.com>
CommitDate: Tue Jan 31 16:03:33 2023 +0000

ntdll: add function wine_get_package_string()
---

 dlls/ntdll/ntdll.spec    | 1 +
 dlls/ntdll/unix/loader.c | 8 ++++++++
 libs/wine/wine.map       | 1 +
 3 files changed, 10 insertions(+)


diff --git a/dlls/ntdll/ntdll.spec b/dlls/ntdll/ntdll.spec
index 248ae72a011..2faca48651f 100644
--- a/dlls/ntdll/ntdll.spec
+++ b/dlls/ntdll/ntdll.spec
@@ -1622,4 +1622,5 @@
 
 # Version
+@ cdecl -syscall wine_get_package_string()
 @ cdecl -syscall wine_get_version()
 @ cdecl -syscall wine_get_build_id()
diff --git a/dlls/ntdll/unix/loader.c b/dlls/ntdll/unix/loader.c
index 676027fe516..a3b13fea10a 100644
--- a/dlls/ntdll/unix/loader.c
+++ b/dlls/ntdll/unix/loader.c
@@ -379,4 +379,12 @@ const char * CDECL wine_get_version(void)
 }
 
+/*********************************************************************
+ *                  wine_get_package_string
+ */
+const char * CDECL wine_get_package_string(void)
+{
+    return PACKAGE_STRING;
+}
+
 
 /*********************************************************************
diff --git a/libs/wine/wine.map b/libs/wine/wine.map
index f272a801743..2ec107930b3 100644
--- a/libs/wine/wine.map
+++ b/libs/wine/wine.map
@@ -93,4 +93,5 @@ WINE_1.0
     wine_get_ss;
     wine_get_user_name;
+    wine_get_package_string;
     wine_get_version;
     wine_init;
--
